apps/test/solutions/applab/ec_canvas.js

```diff
@@ -125,6 +125,59 @@ module.exports = {
         result: true,
         testResult: TestResults.FREE_PLAY
       }
+    },
+
+    {
+      description: 'receives movementX and movementY in mousemove events',
+      editCode: true,
+      xml:
+        'createCanvas("canvas1", 320, 450);' +
+        'textLabel("movementX", "");' +
+        'textLabel("movementY", "");' +
+        'onEvent("canvas1", "mousemove", function(event) {' +
+        '  console.log("movementX: " + event.movementX + " movementY: " + event.movementY)' +
```

Comment:
So my strategy has been to minimize how often we use console.log in unit tests, just because it does show up in console output. What I've tried to do instead is to create another element, and call setText (and later getText) on that element.